src/main/java/spoon/reflect/CtModel.java

```diff
@@ -41,6 +42,17 @@
 	/** returns all packages of the model */
 	Collection<CtPackage> getAllPackages();
 
+	/**
+	 * Returns the unnamed module.
+	 * Warning: if there are other modules, they are not contained in that unnamed root module, use getAllModules instead.
```

Comment:
Is it still true? I guess `getUnnamedModule().getAllModules()` now works same like Model.getAllModules(). OR?

Reply:
No I think you're right.